Source/Windows10/Prism.Windows/Services/Gesture/GestureService.cs

```diff
@@ -8,51 +8,18 @@
 using Windows.UI.Core;
 using Windows.UI.Input;
 using Windows.UI.Xaml;
+using System.ComponentModel;
 
 namespace Prism.Services
 {
-    public class GestureService : IGestureService
+    public class GestureService : IGestureService, IDestructibleGestureService
```

Comment:
Why have a new interface for Destroy that is specific to the gesture service? Just bake it part of the IGestureServcie